docs/conf.py

```diff
@@ -0,0 +1,8 @@
+master_doc = "index"
+project = "sphinxext-rediraffe"
+
+exclude_patterns = ["_build", "Thumbs.db", ".DS_Store"]
```

Comment:
I don't think excluding anything but _build is necessary, right?

Reply:
yeh this is what we use in our excutablebooks packages, but just `_build` should generally be fine.